src/SurfingImGui/SurfingGuiManager.cpp

```diff
@@ -3147,6 +3147,7 @@ void SurfingGuiManager::EndWindowSpecial(int index) {
 void SurfingGuiManager::EndWindow() {
 	rectangleLastWindowShape = getWindowShape();
 
+  ImGui::Dummy(ImVec2(0, 0)); // see ImGui::ErrorCheckUsingSetCursorPosToExtendParentBoundaries
```

Comment:
This is the "fix" for the assertion failure I mentioned: see the long comment in the imgui method for details: this might be a real fix, but I'm not sure.